backend/Squirrel.ConsoleApp.BL/Services/ConnectionFileService.cs

```diff
@@ -0,0 +1,42 @@
+﻿using Newtonsoft.Json;
+using Squirrel.ConsoleApp.BL.Interfaces;
+using Squirrel.ConsoleApp.Models.Models;
+
+namespace Squirrel.ConsoleApp.BL.Services;
+
+public class ConnectionFileService: IConnectionFileService
+{
+    public void CreateEmptyFile()
+    {
+        var filePath = GetFilePath();
+        if (!File.Exists(filePath))
+        {
+            File.WriteAllText(filePath, "{}");
+        }
+    }
+
+    public ConnectionString ReadFromFile()
+    {
+        var filePath = GetFilePath();
+        if (File.Exists(filePath))
+        {
+            var json = File.ReadAllText(filePath);
+            return JsonConvert.DeserializeObject<ConnectionString>(json) ?? throw new InvalidOperationException();
+        }
+
+        throw new FileNotFoundException(filePath);
```

Comment:
it's better to throw all exceptions and return from method as fast as possible if errors occured, so use guard clauses:
`if(!File.Exists(filePath) { throw new FileNotFoundException; }
// other code if everything is good`